flask_jsonapi/nested/nested_resource_repositories.py

```diff
@@ -6,8 +6,8 @@
 
 class NestedResourceRepositoryListView(resource_repository_views.ResourceRepositoryViewMixin,
                                        flask_jsonapi.nested.nested_resources.NestedResourceList):
-    def read_many(self, filters):
-        return self.repository.get_list(filters)
+    def read_many(self, filters, pagination):
+        return self.repository.get_list(filters, pagination)
 
```

Comment:
def get_count(self, filters):
        return self.repository.get_count(filters)

Reply:
Added in last commit